src/CarbonAware.CLI/test/EmissionRootCommandUnitTests.cs

```diff
@@ -23,10 +25,15 @@ public void AddEmissionsCommand_CreatesCorrectSubCommandAndOptions()
             EmissionsRootCommand.AddEmissionsCommands(ref command, _aggregator.Object);
 
             var subCommands =  command.Subcommands.AsEnumerable();
-            var listCommand = subCommands.First().Subcommands.First();
 
+            // Find the emissions observed keyword command
+            var observedCommand = subCommands.Where(command => command.Name == "observed").First();
+
+            // We only added the emissions commands so there should be exactly one keyword off the root command
             Assert.That(command.Subcommands.Count, Is.EqualTo(1));
-            Assert.That(listCommand.Options.Count, Is.EqualTo(4));
+
+            // Options are startTime and endTime for the observed keyword
+            Assert.That(observedCommand.Options.Count, Is.EqualTo(2));

```

Comment:
nit: Shouldn't be useful to assert the option 'names'? That way the command is 'tight' to the supported options.

Reply:
Done - improved the readability and correctness of that test.